File: app/src/main/java/androidx/appcompat/app/AppCompatDelegateCompat.java
package androidx.appcompat.app;

import android.annotation.SuppressLint;
import android.content.Context;

import androidx.annotation.NonNull;

public class AppCompatDelegateCompat {
    private AppCompatDelegateCompat() {}

    @SuppressLint("RestrictedApi")
    public static int mapNightMode(@NonNull AppCompatDelegate delegate, @NonNull Context context,
                                   int mode) {
        return ((AppCompatDelegateImpl) delegate).mapNightMode(context, mode);
    }
}

File: app/src/main/java/androidx/appcompat/app/AppCompatDelegateCompat.kt
package androidx.appcompat.app

import android.annotation.SuppressLint
import android.content.Context

object AppCompatDelegateCompat {

    @SuppressLint("RestrictedApi")
    fun mapNightMode(delegate: AppCompatDelegate, context: Context, mode: Int): Int {
        return (delegate as AppCompatDelegateImpl).mapNightMode(context, mode)
    }
}